### ISSUE TITLE
✅ 🐞 Bug Report: Icon appears deformed and title loses alignment if a nested Tab Title spans more than one line

### ISSUE THREAD

**Issue by vgstef** [elementor/elementor#23670]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Using the tab widget, I have titles that are long and get wrapped on a second line.

If I add an icon, the icon on the 2 lines titles gets squeezed and cropped on the sides.

A second problem occurs with the 2 lines headings: it can't get right aligned. They're always left aligned.

See screenshot:
![image](https://github.com/elementor/elementor/assets/5083355/103f0d06-c895-4a3e-b3de-8dcddad909b4)

### Steps to reproduce

1. Add a tab widget
2. Create tabs with different length for the titles.
3. Add an icon

You'll see the icon squeezed/cropped in the headings that have a long title.

4. Right align the title.

You'll see the 2 lines titles don't change.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v5.7.38
	PHP Version: 7.4.33
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 32M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.3.1
	WP Multisite: No
	Max Upload Size: 32 Mo
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: fr-CA
	Timezone: America/Toronto
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.8.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: fr_CA
	User Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36

== Active Plugins ==
	Classic Editor
		Version: 1.6.3
		Author: WordPress Contributors

	Duplicator
		Version: 1.5.5.1
		Author: Duplicator

	DynamicConditions
		Version: 1.6.2
		Author: RTO GmbH

	Elementor
		Version: 3.15.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.15.1
		Author: Elementor.com

	LiteSpeed Cache
		Version: 5.6
		Author: LiteSpeed Technologies

	Pods - Custom Content Types and Fields
		Version: 2.9.19
		Author: Pods Framework Team


== Fonctionnalités ==
	Custom Fonts: 0
	Custom Icons: 0

== Intégrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive
	Editor Top Bar: Inactive by default
	Landing Pages: Inactive
	Éléments imbriqués: Active
	Lazy Load Background Images: Active
	Global Style Guide: Inactive by default
	Page Transitions: Active
	Notes: Inactive
	Boucle: Active
	Envois de formulaire: Active
	Scroll Snap: Inactive
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default


				

````

</details>

**Comment by nicholaszein** [comment]
Hello @vgstef! 

Thank you for submitting your issue! 🙏 

🏷️ We've labeled this issue and our team will take a look at it. If there is indeed a problem, they'll work on fixing it. We can't say exactly when it'll be sorted out, but we kindly ask for your patience. Thanks for understanding!

Best regards

**Comment by nicholaszein** [comment]
Hello there @vgstef!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Core v3.16.3! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
